### PR TITLE
awards categories show winner, translations, latest kibble

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
This PR:

- updates to the latest kibble which includes award category winner bool
- shows winner_of instead of nominated_for on detail page if IsWinner
- adds winner_of i18n translations

Card here: [AB#6111](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Web%20team/Stories/?workitem=6111)
also [AB#6479](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Web%20team/Stories/?workitem=6479)

### CODE REVIEW

**Comment by matthewthomsonnz** [comment]
@brynyShift72 @l0ud0gg @curtisadams28 pls

File: site/es_MX.all.json [hunk 1]
```diff
@@ -409,5 +409,6 @@
 
   "app_badge_title": { "other": "Download the app to view your purchased content!" },
   "app_badge_ios": { "other": "Download on the App Store" },
-  "app_badge_android": { "other": "Get it on Google Play" }
+  "app_badge_android": { "other": "Get it on Google Play" },
+  "awards_winner_of": { "other": "Ganador de" }
```
Comment (l0ud0gg) [inline site/es_MX.all.json:413]: looks like mx might be missing the other awards keys? in competition / nominated for

**Review by l0ud0gg — CHANGES_REQUESTED** [review]
couple of missing keys for MX language by the looks. otherwise looks fine. maybe once its released we can deploy latest core template to the https://tvoddemo-addons.shift72.com/ site with all the languages to flick through them all and see them on the frontend.

**Comment by stajs** [comment]
@matthewthomsonnz Just acknowledging the @mention. I wasn't a reviewer but looks fine to me, with just @l0ud0gg comment about MX.

**Comment by additiverich** [comment]
@l0ud0gg Where are the MX translations coming from?

**Comment by stajs** [comment]
@additiverich At the risk of seeming ignorant, [they seem to be mostly the same](https://www.getblend.com/blog/is-mexican-a-language/).

**Review by stajs — APPROVED** [review]

**Pull request merged**